fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -2650,6 +2657,10 @@ private boolean createOlapTable(Database db, CreateTableStmt stmt) throws UserEx
             } catch (AnalysisException e) {
                 throw new DdlException(e.getMessage());
             }
+            if (enableUniqueKeyMergeOnWrite && !enableLightSchemaChange && !CollectionUtils.isEmpty(
+                    keysDesc.getClusterKeysColumnNames())) {
+                throw new DdlException("Unique merge-on-write table with cluster keys must enable light schema change");
```

Comment:
Unique merge-on-write tables with cluster keys require light schema change to be enabled.